pmm/pmm-documentation.groovy

```diff
@@ -0,0 +1,54 @@
+pipeline {
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'publish',
```

Comment:
you want to use the publish branch for pushing as default?

Reply:
Yes.